src/pkg/quota/quota.go

```diff
@@ -20,19 +20,19 @@ type Quotas struct {
 
 func (q Quotas) Validate(service *defangv1.Service) error {
 	if service.Name == "" {
-		return errors.New("service name is required") // CodeInvalidArgument
```

Comment:
is the colon suppose to be there? (below also).

Reply:
I added it because I thought it'd be more clear. Not sure.